src/components/Header.js

```diff
@@ -0,0 +1,42 @@
+import React from 'react';
+import { Nav } from 'reactstrap'
+import { NavLink } from "react-router-dom"
+import Logo from "./../assets/Logos/pmlLarge.png"
+
+
+
+const Header = () => {
+
+
+
```

Comment:
This component needs a little bit of formatting. Looks like there are a lot of extra line breaks and inconsistent quotations.